conf/map/battle/pet.conf

```diff
@@ -57,12 +57,19 @@ pet_attack_support: false
 // When the master receives damage from the monster, whether or not the pet attacks back. (Note 1)
 pet_damage_support: false
 
-// Minimum intimacy necessary for a pet to support their master. Default is 900
-// (intimacy goes from 0 to 1000). At this minimum, support rate is 50% of pet's normal value.
+// Minimum intimacy necessary for a pet to support their master. Default is 900 for pre-RE and 910 for RE.
+// (Intimacy goes from 0 to 1000.) At this minimum, support rate is 50% of pet's normal value.
 // At max (1000) support rate is 150%.
+// Min value: 0
+// Max value: 950
```

Comment:
2&3 lines above, max is written as 1000, so why this is limited till 950?

Reply:
The max value for intimacy is 1000. The max value for these two settings is 950.
I actually don't know, why it's capped to 950, but that's nothing I've changed. I just wrote that note to the setting descriptions.

Reply:
Okay, I did some research. These settings' max value needs to be less than 1000 because the calculation of `pd->rate_fix` would end up in a devided by 0 error otherwise.
`status_calc_pet_()`:
```pd->rate_fix = 1000*(pd->pet.intimate - battle_config.pet_support_min_friendly)/(1000- battle_config.pet_support_min_friendly) +500;```